clover_blocks/www/blocks.js

```diff
@@ -0,0 +1,481 @@
+const COLOR_FLIGHT = 293;
+const COLOR_STATE = 36;
+const COLOR_LED = 143;
+const COLOR_GPIO = 200;
+const DOCS_URL = 'https://clover.coex.tech/en/blocks.html';
```

Comment:
This probably won't be accessible when we are connected to the drone; maybe we should reference a local copy?

Reply:
Reasonable. But can we rely this package is used on Clover's image/official VM but not autonomously?

`selfcheck.py` links to the external site.